schemas.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional
+from datetime import date
+
+from pydantic import BaseModel
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: Optional[str] = None
+    publication_date: date
+
+
+class BookCreate(BookBase):
+    author_id: int
+
+
+class Book(BookBase):
+    id: int
+    author_id: int
+
+    class Config:
+        orm_mode = True
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: Optional[str] = None
+
+
+class AuthorCreate(AuthorBase):
+    pass
+
+
+class Author(AuthorBase):
+    id: int
+    books: list[Book] = []

```

Comment:
The `books` field in the `Author` schema is initialized as an empty list. This is appropriate for representing the relationship, but ensure that it does not lead to unintended behavior if not all authors have books.